docs/_articles/en/user-guide/org-browser.md

```diff
@@ -38,20 +38,22 @@ You can refresh metadata at org level, for a type, for folders in a type, and fo
 - folder metadata type to update the folders in the type, but not the components in the folder
 - folder in a metadata type to refresh the components in it
 
-![Metadata type with folders](./images/org_browser_folders.png)
+![Metadata type with folders](../../../images/org_browser_folder_ret.png)
 
 ## Retrieving Metadata
 
 You can retrieve a single metadata component or all components of the same type. Click retrieve icon next to:
 
 - component name **(7)** to retrieve a component to your local project
-- metadata type **(8)** to retrive all components of the type
+- metadata type **(8)** to retrieve all components of the type
+
+You can retrieve all folders of folder metadata types such as Reports, Dashboards, Documents, and EmailTemplates, by clicking on the retrieve icon next to the metadata type. When you retrieve a folder, only information about the folder is retrieved  and added to a <folder>-meta.xml file. To retrieve components within folders, view the components in a folder by expanding it and then retrieve the individual component by clicking the retrieve icon. 
 
 Currently, the source is retrieved into your default package directory as defined in the [sfdx-project.json](./en/getting-started/first-project#the-sfdx-projectjson-file) file.
 
 You can also retrieve multiple components from the default org by clicking the retrieve button next to the metadata type. If retrieving a component overwrites it, you'll be prompted to select how to proceed.
 
-![Overwrite components](./images/overwrite-prompt.png)
+![Overwrite components](../../../images/overwrite-prompt.png)
 
 When you retrieve components for a metadata type, the Org Browser automatically refreshes the component list for the selected type and then retrieves them. This ensures that the extensions accurately check the local workspace for existing components.
```

Comment:
For localization reasons and for ensuring clarity regarding the antecedent, not supposed to use "This" without a noun. like: This action ensures... or Org Browser ensures that the ...

Reply:
Is the sentence that follows supposed to be a note?

Reply:
> Is the sentence that follows supposed to be a note?

It should be a note. Making it so.